docs/src/pages/components/columns.mdx

```diff
@@ -3,15 +3,27 @@ title: Columns
 group: Layout
 github: https://github.com/marigold-ui/marigold/blob/main/packages/components/src/Columns/Columns.tsx
 edit: https://github.com/marigold-ui/marigold/blob/main/docs/src/pages/components/columns.mdx
+caption: Component with sized columns in one row.
 ---
 
-Columns component with sized columns in one row. Add an array of numbers to set the size of the children. The columns array length and the count of children must be the same. You can add space between the columns und set a collapseAt prop to collapse the columns at a certain width.
+The `<Columns>` is a responsive layout component, with sized columns in one row.
+Add an array of numbers to set the size of the children. The columns array length and the count of children must be the same. You can add space between the columns und set a collapseAt prop to collapse the columns at a certain width.
```

Comment:
maybe "with the `columns` property you can set with an array of numbers the size of the children" ? collapseAt also in the `` so it is clear that this is also code?